scopus/affiliation_search.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, query, refresh=False, download=True, count=200):
         download : bool (optional, default=True)
             Whether to download results (if they have not been cached).
 
+        verbose : bool (optional, default=False)
+            Whether to print a downloading progress bar to terminal. requiers download=True.
```

Comment:
Capital letter after full stop. Maybe it's better to write "Has no effect for download=False".  It sounds less harsh to me.